joomla/joomla-cms
# [4.0] Group banners buttons

### Summary of Changes
Group banners buttons in the toolbar


### Testing Instructions
- Apply patch
- Check publish/unpublish/trash/archive functionallity


### Expected result
Works as before
